assets/scss/modules/hero.scss

```diff
@@ -11,9 +11,25 @@
   }
 }
 
-.strike::before,
-  .strike::after {
-    background-color: $picton-blue;
-    height: 1rem;
-    width: 37.5rem;
-  }
+.line {
+  background-image: url("../img/line.png");
+  margin-left: 12.5rem;
+  min-height: 2px;
+  width: 37.5rem;
+}
+
+.stay-button {
+  background: rgba(0, 0, 0, 0.0);
+  border: 1px solid $picton-blue;
+  border-radius: 5px;
+  color: $white;
+  font-size: 18px;
+  margin: 3.125rem auto;
+  min-width: 11.5rem;
```

Comment:
Properties should be ordered background, border, border-radius, color, font-size, margin, min-height, min-width, overflow